hedera-node/hedera-app/src/xtest/java/contract/ClassicViewsXTestConstants.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (C) 2023 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package contract;
+
+import com.esaulpaugh.headlong.abi.Function;
+import com.hedera.hapi.node.base.ContractID;
+import com.hedera.hapi.node.base.FileID;
+import com.hedera.pbj.runtime.io.buffer.Bytes;
+
+public class ClassicViewsXTestConstants {
+    static final FileID CLASSIC_VIEWS_INITCODE_FILE_ID = new FileID(0, 0, 1029L);
```

Comment:
(Where does 1029 here and 1030 on the next line come from? Is it that we know in what order we create things, and we always start at 1028, or what?)

Reply:
The xtest initialization methods create a bunch of entities including the file for the contract bytecode.  It is assumed that 30 entities are enough because `MiscViewsXTestConstants.NUM_ENTITY_NUM = 1030L` and this demarks the first entity number created during the test.  The pattern is just to use 29 for the initcode file as I think this is the last entity that needs to be pre-created.